File: JAVA/Day2/TaskThree.java
public class TaskThree {
public String likeSystem(String[] array){
String output = "";
if (array.length == 0){
output = "no one likes this";
}
if (array.length == 1){
output = array[0] + " likes this";
}
if (array.length == 2){
output = array[0] + " and " + array[1] + " likes this";
}
if (array.length == 3){
output = array[0] + ", " + array[1] + " and " + array[2] + " likes this";
}
if (array.length > 3){
output = array[0] + ", " + array[1] + " and " + (array.length - 2) + " others like this";
}
return output;
}


public static void main(String[] args){
TaskThree object = new TaskThree();
String[] myArray ={"Max", "John", "Mark"};
String result = object.likeSystem(myArray);
System.out.println(result);
 
}

}

File: PYTHON/Day2/TaskThree.py
def like_system(list):
	output = ""
	if isinstance(list, (int, float)):
		raise ValueError("list must be a string")
	if (len(list) == null):
		output = "no one likes this"
		raise ValueError("list must be a string")
		
	if (len(list) == 1): 
		output = list[0]+" likes this"
		
	if (len(list) == 2): 
		output = list[0]+" and "+ list[1]+" likes this"
	if (len(list) == 3):
		output = list[0]+", "+list[1]+" and " + list[2]+" likes this"
	if (len(list) > 3): 
		output = list[0]+", "+list[1]+" and "+str((len(list) - 2))+" others like this"
	return output
	
	
mylist = ["Max", "John", "Mark", "John"]
print(like_system(mylist));
